app/assets/stylesheets/sequence_builder.css

```diff
@@ -165,15 +167,6 @@ padding-left: 10px;
 .content-area{
   padding-top:15px;
 }
-
-/* BUTTONS */
-
-.green-button{
```

Comment:
Are these replacements for the existing classes (Eg: `red`, `yellow`, `green` etc)?

If they are replacements, we can go ahead and remove the older, more neon-y versions.